src/cosmos/auth/v1beta1/query.ts

```diff
@@ -357,76 +399,3 @@ export const QueryParamsResponse = {
     return message;
   },
 };
-
-/** Query defines the gRPC querier service. */
-export interface Query {
```

Comment:
What's the reason for moving this into a separate file? Isn't it nicer (for now) to keep 1 output file for each input file?